crates/sui-source-validation-service/src/lib.rs

```diff
@@ -316,7 +320,31 @@ pub async fn initialize(config: &Config, dir: &Path) -> anyhow::Result<NetworkLo
         }
     }
     clone_repositories(repos, dir).await?;
-    verify_packages(config, dir).await
+    let sources = verify_packages(config, dir).await?;
+    let sources_list = sources_list(&sources).await;
+    Ok((sources, sources_list))
+}
+
+pub async fn sources_list(sources: &NetworkLookup) -> NetworkLookup {
```

Comment:
I'm opting to construct the sources list at startup rather than, e.g., on request. It seemed easiest to just build it up from the existing `sources` and `SourceInfo` type and set the `source` to none.